docs/adding games.md

```diff
@@ -1,269 +1,76 @@
+it will not be detailed here.
+
+## Client
+
+The client is an intermediary program between the game and the Archipelago server. This can either be a direct
+modification to the game, an external program, or both. This can be implemented in nearly any modern language, but it
+must fulfill a few requirements in order to function as expected. The specific requirements for the game client in
+order to behave as expected are as follows:
+
+* Handle both ws and wss connections
+* Detect and react when a location has been "checked" by the player by sending a network packet to the server
+* Receive and parse network packets when the player receives an item from the server, and reward it to the player on
+demand
+  * **Any** of your items can be received any number of times, up to and far surpassing those that the game might
+normally expect
+  * Items can be sent from the server and so might not have a player or location attributed to them
+* Be able to change the port for saved connection info
+  * Rooms hosted on the website attempt to reserve their port, but since there are a limited amount of numbers, this
+privilege can be lost, requiring the room to be moved to a new port
+* Reconnect if the connection is unstable and lost while playing
+* Keep an index for items received in order to resync. Order is guaranteed.
+* Received items that were sent to the player while they were not connected to the server
+  * The player being able to complete checks and sending them when reconnecting is a good bonus but not strictly 
+required
+* Send a status update packet alerting the server that the player has completed their goal
+
+Libraries for most modern languages and the spec for various packets can be found in the
+[network protocol](/docs/network%20protocol.md) API reference document.
+
+## World
+
+The world is your game integration for the Archipelago generator, webhost, and multiworld server. It contains all the
+information necessary for creating the items and locations to be randomized, the logic for item placement, the 
+datapackage information so other game clients can recognize your game data, and documentation. Your world must be
+written as a Python package to be loaded by Archipelago. This is currently done by creating a fork of the Archipelago
+repository and creating a new world package in `/worlds/`. A bare minimum world implementation must satisfy the 
+following requirements:
+
+* a folder within `/worlds/` that contains an `__init__.py`
+* A `World` subclass where you create your world and define all of its rules
+* A unique game name
+* For webhost documentation and behaviors, a `WebWorld` subclass that must be instantiated in the `World` class 
+definition
+  * the game_info doc must follow the format `{language_code}_{game_name}.md`
+* A mapping for items and locations defining their names and id's for clients to be able to identify them, 
```

Comment:
Instead of a comma, these should be broken apart either into two sentences or with something like parentheses or —
```suggestion
* A mapping for items and locations defining their names and id's for clients to be able to identify them. These are
```